content/posts/rh-distributed-tracing/index.mdx

```diff
@@ -0,0 +1,236 @@
+
+The presence of an OpenTelemetryCollector in a namespace (project) triggers the creation of several resources in that namespace
+including a deployment of the OpenTelemetry Collector. In simple terms, all that is required to launch an OTLP collector in your
+namespace is the creation of an OpenTelemetryCollector. This custom resource configures OTLP receivers and exporters. An OTLP collector is an
+intermediary that allows OTLP data to be funnelled to a variety of backends. Traces, logs, and metrics can be exported from an application with a
+single OTLP collector.
+
+The presence of an `Instrumentation` custom resource triggers the automatic instrumentation of an application. This is available for a
+limited list of languages. At this time Python, Java, and Ruby offer auto-instrumentation. For more information, visit the
+[automatic instrumentation docs](https://opentelemetry.io/docs/instrumentation/java/automatic/).
+
+Visit the Red Hat Distributed Tracing Data Collection
+[documentation](https://catalog.redhat.com/software/operators/detail/615486a469cb9f1af5ba7421) and the OpenTelemetry Operator project
+on [GitHub](https://github.com/open-telemetry/opentelemetry-operator) for additional details and examples.
+
+### Red Hat Distributed Tracing Platform
+
+The Red Hat Distributed Tracing Platform is an opinionated Jaeger Operator. This operator watches all namespaces in the `Smaug cluster` for any
+namespaced `Jaeger` custom resources. The presence of a Jaeger in a project triggers the creation of several
+resources including a deployment of Jaeger along with a route to the Jaeger frontend. The example below creates an `AllInOne` instance
+of Jaeger that uses in-memory storage. For production modes, visit the Red Hat Distributed Tracing Platform
+[documentation](https://catalog.redhat.com/software/operators/detail/5ec54a5c78e79e6a879fa271). Jaeger is a tool to analyze traces. There are many
+vendors and open source projects that offer backends for traces, logs, and metrics - all of which can be funnelled through a single OTLP collector.
+The [OpenTelemetry Registry](https://opentelemetry.io/registry/) lists OTLP compatible tools.
+
+### Example Application
+
+For this blog, an [example golang application](https://github.com/sallyom/golang-ex/tree/master/opentelemetry-sample-app) is deployed
+in the Smaug cluster `otel-dev` project. The application includes a simple web server, a service that exposes port 8080 of the web
+server pod, and a route that exposes the service. View the single yaml file
+[here](https://raw.githubusercontent.com/sallyom/golang-ex/master/opentelemetry-sample-app/sample-app.yaml).
+To deploy this application in your own cluster, first create an `otel-dev` namespace or project (or, substitute `otel-dev` everywhere for your namespace).
+and then apply the sample-app yaml file. Since Smaug is an OpenShift cluster, here we use the command-line tool `oc`. However,
```

Comment:
```suggestion
To deploy this application in your own cluster, first create an `otel-dev` namespace or project (or, substitute `otel-dev` everywhere for your namespace)
and then apply the sample-app yaml file. Since Smaug is an OpenShift cluster, here we use the command-line tool `oc`. However,
```